### ISSUE TITLE
What is this state of this project?

### ISSUE THREAD

**Issue by ryanwinchester** [zyro/elixir-uuid#50]
I see the maintainer is not responding to issues or PRs for like two years.

Are there any plans to keep maintinaing this @zyro?

Or is there anybody who will inherit it or take over?

There are useful PRs getting ignored, then closed by OP, like https://github.com/zyro/elixir-uuid/pull/45

**Comment by leifg** [comment]
Following

**Comment by yordis** [comment]
@zyro it seems that you are focused on Golang at the moment and don't much time for this package.

Would you mind transferring the ownership or adding contributors that would help you to maintain the package? 

I think the package is popular enough where it grants some love for the greater of the community, you created a super important package for us 🚀 

What do you think?

**Comment by ryanwinchester** [comment]
In the meantime, I've created a fork* that I've been using for the last few months now in production: https://github.com/sevenshores/elixir-uuid-utils
  
I'd love to see a new maintainer on this repo instead, though. Since it's already a dependency on a lot of other projects/libraries.

_* because (a) this one isn't maintained and (b) I wanted UUID v6_

**Comment by jamesvl** [comment]
@ryanwinchester I'm interested in helping out here; message me and I'd love to discuss further what taking our maintenance might look like.

**Comment by zyro** [comment]
@ryanwinchester As you and @yordis have rightly noted, I'm spending my time elsewhere at the moment. This is outside the Elixir space entirely (by necessity, I still love Erlang/Elixir!) and often on closed-source projects. Since I've not directly used this library myself in some time I admit it's basically been in free fall down my priority list.

> it's already a dependency on a lot of other projects/libraries

I agree with what you're saying here, I would like to continue supporting the various projects and packages that depend on elixir-uuid.

I'm open to adding contributors if there are volunteers to maintain this package, and I can push releases to hex when ready. That's as much as I can promise for the foreseeable future.

@ryanwinchester Are you interested in contributing here directly?

**Comment by ryanwinchester** [comment]
> @ryanwinchester Are you interested in contributing here directly?

Yes, that would probably be better for the community than having multiple projects.

Is there a way I can contact you to discuss?

**Comment by yordis** [comment]
@zyro any update on the topic?

**Comment by zyro** [comment]
Had a brief DM exchange with @ryanwinchester but no resolution so far. Will update here if that changes. 👍

**Comment by yordis** [comment]
@zyro is it much to ask to take some of the open PRs, and release a version?

The package is quite stable and probably wouldn't require many changes (hopefully).

**Comment by ryanwinchester** [comment]
FYI:

@bitwalker started a project doing the same things as what I added to my fork of this (at https://github.com/sevenshores/elixir-uuid-utils): v6 support and Ecto type. He's also well-known and respected in Elixir community, so I'll archive mine and use his if it shows like it's gaining traction:

* https://github.com/bitwalker/uniq
* https://github.com/bitwalker/uniq_compat

**Comment by ryanwinchester** [comment]
Re-opening so people can find it
